sdk/component/conf.go

```diff
@@ -163,22 +153,15 @@ func RunnerWithInstanceID(id uuid.UUID) RunnerOption {
 }
 
 // RunnerWithStorer can be used to customise the underlying storage.
-func RunnerWithStorer(stType string, store Storer) RunnerOption {
+func RunnerWithStorer(store Storer) RunnerOption {
 	return func(r *runner) error {
-		switch {
-		case !isAllowedStoreType(storeType(stType)):
-			return ErrRunnerOption{
-				OptionName: "store_type",
-				Reason:     errReasonUnsupportedValue,
-			}
-		case store == nil:
+		if store == nil {
```

Comment:
same comment about Nil check for the storer

Reply:
Done